presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -45,6 +47,39 @@ public Estimate getDistinctValuesCount()
         return distinctValuesCount;
     }
 
+    @Override
+    public String toString()
+    {
+        return "ColumnStatistics {" +
+                "dataSize: " + getDataSize().toString() + ", " +
+                "nullsCount: " + getNullsCount().toString() + ", " +
+                "distinctValuesCount: " + getDistinctValuesCount().toString() + "}";
+    }
+
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        ColumnStatistics columnStatistics = (ColumnStatistics) o;
+        return getDataSize().equals(columnStatistics.getDataSize()) &&
```

Comment:
Is that generated. Generaly (from what I see) Presto uses `Objects.equals` in `equals()` methods. Please regenrate using that.